future/atomicTransactionComposer.go

```diff
@@ -0,0 +1,681 @@
+package future
```

Comment:
The tx composer implements relies on features provided by the future package so it should also be in future until that changes. Otherwise, we'd have code in the transactions package relying on future as a dependency.